docs/adr/0007-record-architecture-decisions.md

```diff
@@ -0,0 +1,19 @@
+# 1. Record architecture decisions
```

Comment:
I clearly need to update the docs re: "how to ADR", because this ADR already exists, as ADR #0001.

In the meantime, could we please delete this file, and rename 0008 to 0007?